ansible/roles/test/tasks/iface_naming_mode/show_pfc_counters.yml

```diff
@@ -0,0 +1,32 @@
+- block:
+    - name: get pfc Rx counter values 
+      shell:  sudo show  pfc  counters | sed   -n '/Port Rx/,/^$/p' 
+      register: pfc_rx
+
+    - debug: var=pfc_rx.stdout_lines
+
+    - name: get pfc Tx counter values
+      shell:  sudo show  pfc  counters | sed   -n '/Port Tx/,/^$/p'
+      register: pfc_tx
+
+    - debug: var=pfc_tx.stdout_lines
+
+    - name: check if the output shows default interface names in default mode 
+      assert:
+        that:
+          - "'{{item}}' in pfc_rx.stdout and '{{item}}' in pfc_tx.stdout "
+          - "'{{port_name_map[item]}}' not in pfc_rx.stdout  and  '{{port_name_map[item]}}' not in  pfc_tx.stdout"
+      with_items: default_interfaces
+      when: mode=='default'
+
+    - name: check if the output shows alias interface names in alias mode 
+      assert:
+        that:
+          - "'{{item}}' in pfc_rx.stdout and '{{item}}' in pfc_tx.stdout "
+          - "'Ethernet' not in pfc_rx.stdout and 'Ethernet' not in  pfc_tx.stdout "
```

Comment:
Checking for the string "Ethernet" is not reliable enough, as some vendor aliases also contain the string "Ethernet". The proper solution is to retrieve the appropriate port name and alias from the device's config and check against them explicitly.

Reply:
modified as per comments